src/scripts/inclusion-bot.yaml

```diff
@@ -495,9 +499,20 @@ triggers:
       - confusing language
     why: >-
       The term ":TERM:" was used to stereotype African languages and cultures
-      that didn't make sense to European colonizers. See [Here are some commonly
-      used terms that actually have racist origins](https://abcnews.go.com/Politics/commonly-terms-racist-origins/story?id=71840410)
-      for more.
+      that didn't make sense to European colonizers.
+
+  - matches:
```

Comment:
clarification: How are dashes handled (e.g. `kow-tow`)? Do they need to be explicitly added or does the parser handle that with these spellings?

Reply:
They need to be manually added